dist/configs/game/maprotation.cfg

```diff
@@ -1,16 +1,22 @@
 rotation1
 {
 	antares
-	chasm
+	if numPlayers > 6 #skip-chasm
```

Comment:
I kind of see 6 as the minimum number of players for the game to be any good... so it is a bit strange to skip for as few as 7 players. 

Also there might be some players idle/spectating.

Reply:
Ok, I've bumped it to some larger value, to skip if `>9` players